platform/vendor/tekton/catalog/main/task/buildpacks-phases/0.2/buildpacks-phases.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     tekton.dev/tags: image-build
     tekton.dev/displayName: "Buildpacks (phases)"
     tekton.dev/platforms: "linux/amd64"
+    tekton.dev/deprecated: "true"
```

Comment:
Something else we need to upgrade?

Reply:
Needs a bit of work. 0.3 cause an admission webhook violation